src/server/game/Entities/AreaTrigger/AreaTriggerTemplate.h

```diff
@@ -164,6 +164,8 @@ class AreaTriggerMiscTemplate
         TimeToTarget        = 0;
         TimeToTargetScale   = 0;
 
+        ExtraScaleCurve     = { 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 1.0000001f, 1.0f };
```

Comment:
AREATRIGGER_EXTRA_SCALE_CURVE + 5: 1065353217/1
AREATRIGGER_EXTRA_SCALE_CURVE + 6: 1/1.401298E-45

that are the defaults